target/linux/ramips/mt7621/base-files/lib/preinit/09_prepare_eeprom

```diff
@@ -0,0 +1,17 @@
+. /lib/functions/system.sh
+
+preinit_prepare_eeprom() {
+	case $(board_name) in
+	netis,n6)
+		EEPROM="/tmp/mt7915_eeprom_dbdc.bin"
```

Comment:
I think we are not packaging `/lib/firmware/firmware/mt7915_eeprom_dbdc.bin`. 
You could copy the patched firmware file here directly to `/lib/firmware/firmware/mt7915_eeprom_dbdc.bin`, then you have to do the patching only once. At the beginning of the script you can check if it already exists.

Reply:
> I think we are not packaging `/lib/firmware/firmware/mt7915_eeprom_dbdc.bin`. 

Exactly.

> You could copy the patched firmware file here directly to `/lib/firmware/firmware/mt7915_eeprom_dbdc.bin`, then you have to do the patching only once. At the beginning of the script you can check if it already exists.
>
> Maybe `target/linux/ramips/mt7621/base-files/etc/hotplug.d/firmware/11-mt76-caldata` would not even be needed in this case.

`/overlay` is not mounted yet (or, at least, not always mounted) when `09_prepare_eeprom` runs. So, I implemented your idea in `11-mt76-caldata`. This allowed us to remove `09_prepare_eeprom`. Thanks.